source/includes/steps-tutorial-ios-swift.yaml

```
@@ -612,11 +612,14 @@ content: |
 
   We read the :term:`partition value` from the {+realm+}. We'll use that when
   creating Tasks. Then we query for the :ref:`collection
-  <ios-client-collections>` of all tasks within that {+realm+}. This collection
-  is live: as we add, remove, and update tasks -- even from another device --
-  this collection always contains the latest results of its associated query.
-  We'll get to reacting to these updates later. First, let's continue setting up
-  the TasksViewController.
+  <ios-client-collections>` of all tasks within that {+realm+}. We sort the
+  Tasks by their ``_id`` property so that they always appear in the same
+  relative order from run to run.
```

```suggestion
  relative order.
```

Nit: this elaboration isn't necessary.